backend/application/syson-sysml-import/src/test/resources/ASTTransformerTest/convertAliasTest/alias.sysml

```diff
@@ -0,0 +1,10 @@
+package AliasImport {
+    package Definitions {
+        part def Vehicle;
+        alias Car for Vehicle;
+    }
+    package Usages {
+        import Definitions::Car;
```

Comment:
Could you please specify the visibility of the import?

Reply:
fait.